pages/ecosystem.html

```diff
@@ -4,19 +4,21 @@
 date: 2014-02-27 05:16:30.000000000 +08:00
 permalink: /ecosystem/
 ---
+
 <div class="row">
 <div class="col-md-2">
-<ul class="nav nav-pills nav-stacked">
-	<li class="active"><a href="#producers" data-toggle="pill">Producers</a></li>
-	<li><a href="#consumers" data-toggle="pill">Consumers</a></li>
-	<li><a href="#liveservices" data-toggle="pill">Live Services</a></li>
-	<li><a href="#sdk" data-toggle="pill">Sample Code</a></li>
-	<li><a href="#tutorials" data-toggle="pill">Tutorials</a></li>
+<ul class="nav nav-pills nav-stacked" target="ecosystem" >
```

Comment:
Target here is used wrongly you can use custom tags which do not change the markup and are not reserved.
e.g. data-target-tab or something of the sort.

Same for the other items in the anchors as the value for target is specified [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attr-target) and some browsers may by default have unspecified behaviours for unknown values here. a target=something implies there is an iframe named something.